Analysis/Producer.py

```diff
@@ -0,0 +1,120 @@
+import ROOT
+import Common.Utilities as Utilities
+import sys
+import os
+
+
+if __name__ == "__main__":
+    sys.path.append(os.environ['ANALYSIS_PATH'])
+
+weights_to_apply = ["weight_Central","weight_tau1_TrgSF_ditau_Central","weight_tau2_TrgSF_ditau_Central", "weight_tauID_Central",]
+files= {
+    "DY":["DY"],
+    "Other":["EWK", "ST", "TTT", "TTTT", "TTVH", "TTV", "TTVV", "TTTV", "VH", "VV", "VVV", "H", "HHnonRes", "TTHH", "ttH"],
+    #"GluGluToBulkGraviton":["GluGluToBulkGraviton"],
+    "QCD":["QCD"],
+    "TT":["TT"],
+    "GluGluToRadion":["GluGluToRadion"],
+    #"VBFToBulkGraviton":["VBFToBulkGraviton"],
+    # "VBFToRadion":["VBFToRadion"],
+    "W":["W"],
+    "data":["data"],
```

Comment:
use `data_unique` - which should be `data.root` on which applied `merge_tuple` filter